<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>Accession Codes &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
<div id="content" role="main" class="columns medium-12 clearfix">
<section id="ae-help" class="columns medium-9">
<h2>Accession Codes</h2>

<ul>
    <li><a href="#four_letter_codes">Full list of four-letter codes</a></li>
    <li><a href="#secondary_acc">Secondary accession numbers</a></li>
</ul>

<p>Experiments and array designs in ArrayExpress are given unique
    accession numbers in the format of</p>
<ul>
    <li>E-XXXX-n for experiments</li>
    <li>A-XXXX-n for array designs</li>
</ul>
<p>XXXX represents a four letter code and n is a number.</p>
<p>E.g.
    E-MEXP-568, A-UHNC-18. Some experiments also have <a
            href="#secondary_acc">secondary accession numbers</a>. </p>

<p> Accessions numbers are generated when sufficient meta-data and data files are provided for a submission. Please refer to our <a href="FAQ.html#need_acc_num_urgent">FAQ</a> for more details about obtaining accession number for your data set.</p>


<p>The four letter code in the accession number indicates the data
    source. The source can be:</p>
<ul>
    <li>ArrayExpress submission tools: <strong>MEXP</strong> for MIAMExpress (deprecated since July 2014), <strong>MTAB</strong> for <a href="/fg/annotare/login">Annotare</a>,
        and <strong>TABM</strong> for Tab2MAGE (deprecated since January 2012).</li>

    <li>Data management tools / submisson pipelines from other research
        organizations (see <a href="#four_letter_codes">full list below</a>).</li>
</ul>
<p><em>Note that the 4 letter code does not necessarily tell you
    which organization performed the experiment or manufactured the array
    design.</em>
</p>


<h3><a id="four_letter_codes">Full list of four-letter codes</a></h3>
<p>Please note that some codes are no longer in active use, i.e. they are not used in accession numbers for new submisisons but are still valid. Some URLs are dead, or point to webpages which are out of maintenance, but are included in the table below for completeness.</p>
<table>
<thead>
<tr>
    <th>Code</th>
    <th>Source</th>
    <th>URL</th>
</tr>
</thead>
<tbody>
<tr>
    <td>AFFY</td>
    <td>Affymetrix</td>
    <td><a href="http://www.affymetrix.com/">www.affymetrix.com</a></td>
</tr>
<tr>
    <td>AFMX</td>
    <td>Affymetrix data sets processed by an
        EBI in-house script </td>
    <td>&nbsp;</td>
</tr>
<tr>
    <td>AGIL</td>
    <td>Agilent</td>
    <td><a href="http://www.agilent.com/">www.agilent.com</a></td>
</tr>
<tr>
    <td>ATMX</td>
    <td>Arabidopsis experiments and array
        designs submitted through
        the At-MIAMExpress submission tool</td>
    <td>tool deprecated</td>
</tr>
<tr>
    <td>BAIR</td>
    <td>Biological Atlas of Insulin Resistance
        (BAIR) project</td>
    <td><a href="http://www.bair.org.uk/">www.bair.org.uk</a></td>
</tr>
<tr>
    <td>BASE</td>
    <td> BASE microarray data management tool</td>
    <td><a href="http://base.thep.lu.se/">base.thep.lu.se</a></td>
</tr>
<tr>
    <td>BIOD</td>
    <td>BioDiscovery microarray data management
        tool</td>
    <td><a href="http://www.biodiscovery.com/">www.biodiscovery.com</a></td>
</tr>
<tr>
    <td>BUGS</td>
    <td>Bacterial Microarray Group at St
        George's, University of
        London (BuG@S)</td>
    <td><a href="http://www.bugs.sgul.ac.uk/">www.bugs.sgul.ac.uk</a></td>
</tr>
<tr>
    <td>CAGE</td>
    <td>Compendium of Arabidopsis Gene
        Expression plant developmental
        time series project</td>
    <td><a
            href="http://www.cagecompendium.org/index.htm">www.cagecompendium.org/index.htm</a>
        and <a href="/microarray/cage">www.ebi.ac.uk/microarray/cage</a></td>
</tr>
<tr>
    <td>CBIL</td>
    <td>Computational Biology and Informatics
        Laboratory at
        University of Pennsylvania</td>
    <td><a href="http://www.cbil.upenn.edu/">www.cbil.upenn.edu</a></td>
</tr>
<tr>
    <td>DKFZ</td>
    <td>German Cancer Research Center</td>
    <td><a href="http://www.dkfz.de/">www.dkfz.de</a></td>
</tr>
<tr>
    <td>DORD</td>
    <td>DDBJ Omics Archive (DOR)</td>
    <td><a href="http://trace.ddbj.nig.ac.jp/dor">http://trace.ddbj.nig.ac.jp/dor</a></td>
</tr>
<tr>
    <td>EMBL</td>
    <td>European Molecular Biology Laboratory</td>
    <td><a href="http://www.embl.org/">www.embl.org</a></td>
</tr>
<tr>
    <td>ERAD</td>
    <td>European
        Read Archive Data (pipeline submission from Wellcome Trust Sanger
        Institute. "European Read Archive" is now renamed the "Sequence Read
        Archive" (SRA) as part of <a href="http://www.insdc.org/">INSDC</a>)<br>
    </td>
    <td><a href="http://www.sanger.ac.uk">www.sanger.ac.uk</a><br>
        <a href="http://nar.oxfordjournals.org/content/early/2010/11/08/nar.gkq1019.long">NAR article on the SRA</a></td>
</tr>
<tr>
    <td>FLYC</td>
    <td>FlyChip Microarray services, Cambridge
        Systems Biology
        Centre, UK</td>
    <td><a href="http://www.flychip.org.uk/">www.flychip.org.uk</a></td>
</tr>
<tr>
    <td>FPMI</td>
    <td>Functional Pathogenomics of Mucosal
        Immunity project</td>
    <td><a
            href="http://www.pathogenomics.ca/fpmi">www.pathogenomics.ca/fpmi</a></td>
</tr>
<tr>
    <td>GEAD</td>
    <td>Genomic Expression Archive (GEA)</td>
    <td><a href="https://www.ddbj.nig.ac.jp/gea/index-e.html">www.ddbj.nig.ac.jp/gea/</a></td>
</tr>
<tr>
    <td>GEHB</td>
    <td>GE Healthcare array designs </td>
    <td><a href="http://www.gehealthcare.com/">www.gehealthcare.com</a></td>
</tr>
<tr>
    <td>GEOD</td>
    <td>NCBI Gene Expression Omnibus (GEO)</td>
    <td><a
            href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</a>.
        See also the <a href="GEO_data.html">How data is imported from GEO</a>.</td>
</tr>
<tr>
    <td>GEUV</td>
    <td>Genetic European Variation in Health and Disease (GEUVADIS), A European Medical Sequencing Consortium</td>
    <td><a
            href="http://www.geuvadis.org/web/geuvadis/home">www.geuvadis.org/web/geuvadis/home</a>.</td>
</tr>
<tr>
    <td>HGMP</td>
    <td>Human Genome Mapping Project Resource
        Centre (now closed)</td>
    <td><a
            href="http://www.geneservice.co.uk/home">www.geneservice.co.uk/home</a></td>
</tr>
<tr>
    <td>IPKG</td>
    <td>Leibniz Institute of Plant Genetics and
        Crop Plant Research
        (IPK-Gatersleben)</td>
    <td><a
            href="http://www.ipk-gatersleben.de/Internet">www.ipk-gatersleben.de/Internet</a></td>
</tr>
<tr>
    <td>JCVI</td>
    <td>J. Craig Venter Institute</td>
    <td><a href="http://www.jcvi.org/">www.jcvi.org</a></td>
</tr>
<tr>
    <td>JJRD</td>
    <td>Johnson &amp; Johnson Pharmaceutical
        Research and Development</td>
    <td><a href="http://www.jnjpharmarnd.com/">www.jnjpharmarnd.com</a></td>
</tr>
<tr>
    <td>LGCL</td>
    <td>LGC Limited</td>
    <td><a href="http://www.lgc.co.uk/">www.lgc.co.uk</a></td>
</tr>
<tr>
    <td>MANP</td>
    <td>Coding of experiment was manually
        prepared by EBI staff</td>
    <td>&nbsp;</td>
</tr>
<tr>
    <td>MARS</td>
    <td>Microarray Analysis and Retrieval
        System (MARS) from Graz
        University of Technology, Institute for Genomics and Bioinformatics</td>
    <td><a href="http://genome.tugraz.at/">genome.tugraz.at</a></td>
</tr>
<tr>
    <td>MAXD</td>
    <td>University of Manchester, Micorarray
        Group maxd software</td>
    <td><a
            href="http://www.bioinf.manchester.ac.uk/microarray/maxd">www.bioinf.manchester.ac.uk/microarray/maxd</a></td>
</tr>
<tr>
    <td>MEXP</td>
    <td>EBI MIAMExpress webform submission tool (deprecated since July 2014)</td>
    <td><a
            href="/miamexpress">www.ebi.ac.uk/miamexpress</a></td>
</tr>
<tr>
    <td>MIMR</td>
    <td>MiMiR data warehouse at the Microarray
        Centre, Clinical
        Sciences Centre, Medical Research Council</td>
    <td><a href="http://www.csc.mrc.ac.uk/">www.csc.mrc.ac.uk</a></td>
</tr>
<tr>
    <td>MNIA</td>
    <td>Laboratory of Genetics, National
        Institute on Aging, <br>
        National Institutes of Health</td>
    <td><a href="http://lgsun.grc.nia.nih.gov/">lgsun.grc.nia.nih.gov</a></td>
</tr>
<tr>
    <td>MTAB</td>
    <td>Experiments in <a href="http://www.fged.org/projects/miame/">MAGE-TAB format</a>, submitted via the MTAB spreadsheet submission tool (retired in September 2014), or via <a href="http://www.ebi.ac.uk/fg/annotare/login">Annotare</a></td>
    <td><a
            href="/fg/annotare/login">http://www.ebi.ac.uk/fg/annotare/login</a></td>
</tr>
<tr>
    <td>MUGN</td>
    <td>Integrated Functional Genomics in
        Mutant Mouse Models as
        Tools to Investigate the Complexity of Human Immunological Disease
        (MUGEN) project</td>
    <td><a href="http://www.mugen-noe.org/">www.mugen-noe.org</a>
        and <a href="/fg/mugen">www.ebi.ac.uk/fg/mugen</a></td>
</tr>
<tr>
    <td>NASC</td>
    <td>European Arabidopsis Stock Centre</td>
    <td><a href="http://arabidopsis.info/">arabidopsis.info</a></td>
</tr>
<tr>
    <td>NCMF</td>
    <td>Netherlands Cancer Institute Central
        Microarray Facility </td>
    <td><a href="http://microarrays.nki.nl/">microarrays.nki.nl</a></td>
</tr>
<tr>
    <td>NGEN</td>
    <td>Nimblegen</td>
    <td><a href="http://www.nimblegen.com/">www.nimblegen.com</a></td>
</tr>
<tr>
    <td>RUBN</td>
    <td>Gerry Rubin's lab</td>
    <td><a
            href="http://www.hhmi.org/research/groupleaders/rubin.html">http://www.hhmi.org/research/groupleaders/rubin.html</a></td>
</tr>
<tr>
    <td>RZPD</td>
    <td>RZPD German Resource Center for Genome
        Research (no long accessible) </td>
    <td><a href="http://www.rzpd.de/">www.rzpd.de (old URL)</a></td>
</tr>
<tr>
    <td>SGRP</td>
    <td>Saccharomyces Genome Resequencing
        Project , Wellcome Trust
        Sanger Institute</td>
    <td><a
            href="http://www.sanger.ac.uk/Teams/Team71/durbin/sgrp">www.sanger.ac.uk/Teams/Team71/durbin/sgrp</a></td>
</tr>
<tr>
    <td>SMDB</td>
    <td>Stanford Micorarray Database (moving to Princeton University, as of January 2013)</td>
    <td><a
            href="http://smd.stanford.edu">http://smd.stanford.edu</a></td>
</tr>
<tr>
    <td>SNGR</td>
    <td>Wellcome Trust Sanger Institute</td>
    <td><a href="http://www.sanger.ac.uk/">www.sanger.ac.uk</a></td>
</tr>
<tr>
    <td>SYBR</td>
    <td>Sybaris Project</td>
    <td><a href="http://www.sybaris-fp7.eu/">http://www.sybaris-fp7.eu/</a></td>
</tr>
<tr>
    <td>TABM</td>
    <td>EBI tab2mage spreadsheet submission
        tool (deprecated since January 2012)<br>
    </td>
    <td><a
            href="/cgi-bin/microarray/tab2mage.cgi">www.ebi.ac.uk/cgi-bin/microarray/tab2mage.cgi</a></td>
</tr>
<tr>
    <td>TIGR</td>
    <td>The Institute for Genomic Research (now
        part of the J. Craig
        Venter Institute)</td>
    <td><a href="http://www.tigr.org/">www.tigr.org</a></td>
</tr>
<tr>
    <td>TOXM</td>
    <td>Toxicogenomics experiments </td>
    <td>&nbsp;</td>
</tr>
<tr>
    <td>UCON</td>
    <td>The Hutchison/MRC Research Centre</td>
    <td><a
            href="http://www.hutchison-mrc.cam.ac.uk/">www.hutchison-mrc.cam.ac.uk</a></td>
</tr>
<tr>
    <td>UHNC</td>
    <td>University Health Network Canada</td>
    <td><a href="http://www.uhn.ca/">www.uhn.ca</a></td>
</tr>
<tr>
    <td>UMCU</td>
    <td>University Medical Center Utrecht</td>
    <td><a href="http://www.umcutrecht.nl/zorg">www.umcutrecht.nl/zorg</a></td>
</tr>
<tr>
    <td>WMIT</td>
    <td>Whitehead Institute for Biomedical
        Research/ Massachusetts
        Institute of Technology</td>
    <td><a href="http://www.wi.mit.edu/">www.wi.mit.edu</a></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<h3><a id="secondary_acc"></a>Secondary accession numbers </h3>
<p>Some experiments or array designs have a secondary accession number which links
    to another ArrayExpress experiment/array design, or to an external data source. The
    secondary accession number is shown in the "Links" section on an experiment's page.
    For example, for experiment
    <a href="/arrayexpress/experiments/E-GEOD-42281">E-GEOD-42281</a>:<br>
</p>
<p><img src="acc_code_expt_sec_acc_example.png" alt="Expt Secondary Accession Numbers example" style='border:2px solid #385930'
        ><br>
</p>

<p>For array designs, the secondary accession number can be found in the header of the
    actual design file (in tab-delimited text format), e.g. for array design
    <a href="/arrayexpress/files/A-GEOD-9349/A-GEOD-9349.adf.txt">A-GEOD-9349</a>:<br>
</p>
<p><img src="acc_code_adf_sec_acc_example.png" alt="ADF Secondary Accession Numbers example"
        style='border:2px solid #385930'><br>
</p>

<h4>Common reasons for secondary accession numbers</h4>
<ol>
    <li style="margin-left: 1em;"><em>Data re-analysis</em>: if the data provided is a re-analysis
        of another dataset, the
        accession number of the original experiment will be the
        secondary accession number</li>
    <li style="margin-left: 1em;"><em>NCBI GEO-imported data</em>: experimental data imported from the <a
            href="http://www.ncbi.nlm.nih.gov/geo/">NCBI Gene Expression Omnibus
        (GEO)</a> will have
        the GEO series (prefix "GSE") or data set (prefix "GSD") identifier as the secondary accession
        number. Similarly, array design will have the GEO platform (prefix "GPL") accession
        number. The ArrayExpress primary accession and GEO secondary accession for
        GEO-imported data are intuitively correlated. E.g. GEO accession "GSE12345"
        would be associated with ArrayExpress accession "E-GEOD-12345"
        . See also <a href="GEO_data.html">How data is imported from GEO</a>.</li>
    <li style="margin-left: 1em;"><em>High-throughput sequencing experiments</em>: they will have a
        link to the <a href="/ena/">European Nucleotide
            Archive</a> (ENA)
        or <a href="/ega/">European Genome-phenome Archive</a>
        (EGA) where the raw data files (e.g. fastq reads) are kept.</li>
</ol>
</section>
<aside class="columns medium-3 last">
    <h3>Help</h3>
    <ul id="secondary-nav">
        <li><a href="index.html">Help home</a></li>
        <li><a href="quick_start.html">Quick Start guide</a></li>
        <li><a href="how_to_search.html">How to search</a>
            <ul>
                <li><a href="how_to_search_adf.html">Array design search</a></li>
                <li><a href="how_to_search_private_data.html">Search for private data</a></li>
            </ul>
        </li>
        <li><a href="submissions_overview.html">How to submit</a>
            <ul>
                <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
                <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                <li><a href="single-cell_submission_guide.html">Single-cell submissions</a></li>
                <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
            </ul>
        </li>
        <li><a href="data_availability.html">Learn more about our data</a>
            <ul>
                <li><a href="data_availability.html">Data access policy</a></li>
                <li class="active"><a href="accession_codes.html">Accession codes</a></li>
                <li><a href="GEO_data.html">GEO data</a></li>
                <li><a href="magetab_spec.html">MAGE-TAB format</a></li>
            </ul>
        </li>
        <li><a href="programmatic_access.html">Other ways to get data</a>
            <ul>
                <li><a href="programmatic_access.html">Programmatic access</a></li>
				<li><a href="FTP_archives.html">FTP archives</a></li>
				<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
            </ul>
        </li>
        <li><a href="FAQ.html">FAQ</a></li>
        <li><a href="contact_us.html">Contact us</a></li>
    </ul>
</aside>
</div>
</body>
</html>
