<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
    <title>FAQ &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
<div id="content" role="main" class="columns medium-12 clearfix">
    <section id="ae-help" class="columns medium-9">
        <h2>Frequently Asked Questions (FAQ)</h2>
        <p>Please contact us using the "Feedback" button at the top of this page if you cannot find the answer to your
            question.</p>

        <a id="use_ae_faq"></a>
        <h3>1. Using ArrayExpress</h3>
        <h5>Searching and understanding data</h5>
        <ul>
            <li><a href="#search_help">How do I search the ArrayExpress database?</a></li>
            <li><a href="#explain_browse_page">Can you guide me through the table of experiments I get after searching
                ArrayExpress?</a></li>
            <li><a href="#explain_magetab">How do I link sample and data file information in the downloadable files?</a>
            </li>
            <li><a href="#expt_already_published">I read a paper that referred to data in ArrayExpress but when I
                searched
                for the data I got no hits. Why?</a></li>
            <li><a href="#acc_num_prefix">I have seen ArrayExpress experiment accessions with prefixes such as
                "E-MTAB", "E-GEOD", etc. What do the prefixes mean?</a></li>
        </ul>

        <h5>Data access policy - permissions and restrictions</h5>
        <ul>
            <li><a href="#data_restrictions">Are there any restrictions on the use of microarray data obtained through
                ArrayExpress?</a></li>
            <li><a href="#need_login_or_not">Do I need a login account to view or access data in ArrayExpress?</a></li>
            <li><a href="#lost_login_details">I've lost my ArrayExpress website login details. What do I do?</a></li>
            <li><a href="#reviewer_need_access">I'm reviewing a paper. How do I get a login to view private data
                that the authors have deposited in ArrayExpress?</a></li>
            <li><a href="#private_access">Will I have full access to an experiment while it is private in
                ArrayExpress?</a></li>
        </ul>

        <h5>FTP data download and programmatic access</h5>
        <ul>
            <li><a href="#ftp_dl">Is there an FTP site to download data files in bulk?</a></li>
            <li><a href="#standard_url">Is there a standard URL format for linking to an experiment in ArrayExpress?</a>
            </li>
            <li><a href="#api">Do you have any application programming interfaces (APIs) for accessing ArrayExpress?</a>
            </li>
        </ul>

        <h5>Other general questions</h5>
        <ul>
            <li><a href="#cite">How do I cite my ArrayExpress data sets in my publication?</a></li>
            <li><a href="#ae_geo_overlap">How much overlap is there between ArrayExpress and the NCBI Gene Expression
                Omnibus (GEO)?</a></li>
            <li><a href="#update_frequency">How often is ArrayExpress updated?</a></li>
        </ul>

        <a id="submit_faq_general"></a>
        <h3>2. Submitting data to ArrayExpress</h3>

        <h5>Accession numbers</h5>
        <ul>
            <li><a href="#need_acc_num_urgent">I urgently need an accession number to include in my publication - how
                can I get one?</a></li>
            <li><a href="#how_long_to_get_acc">After I've submitted my experiment, how long will it take to get an
                accession number?</a></li>
        </ul>

        <h5>Keeping unpublished data private</h5>
        <ul>
            <li><a href="#keep_private">Can my data be kept private after I submit it?</a></li>
            <li><a href="#journal_private_access">The journal I am submitting to have requested private access to my
                data - how do I get an ArrayExpress login for them?</a></li>
            <li><a href="#double_blind">The journal I am submitting to supports "double-blind" peer review. Can
                ArrayExpress hide my name and contact details from the reviewers?</a></li>
            <li><a href="#private_login_problem">I tried to login to ArrayExpress to see some recently submitted private
                data but it said my username and password were invalid. Why?</a></li>
        </ul>

        <h5>Public release date</h5>
        <ul>
            <li><a href="#what_date">I don't know when my paper will be published so what should I put as the release
                date?</a></li>
            <li><a href="#make_public">My paper is about to be published - how do I make the data public?</a></li>
            <li><a href="#add_publication">My paper has been published - how do I add the citation details to my
                experiment?</a></li>
        </ul>

        <h5>Which files to submit</h5>
        <ul>
            <li><a href="#data_file">Which data files do I need to submit for an experiment?</a></li>
            <li><a href="#mageml">I have MAGE-ML/MAGE-TAB files generated from in-house pipelines. Does ArrayExpress
                still accept them?</a></li>
        </ul>

        <h5>Making changes to data already submitted to Arrayexpress</h5>
        <ul>
            <li><a href="#hit_submit_too_soon">I hit "Submit" at the end of my submission but I haven't actually
                finished editing. I logged in to the submission tool but can't edit my experiment anymore. Help!</a>
            </li>
            <li><a href="#data_already_in_AE_DB">My data has been loaded into ArrayExpress but I need to change/correct
                something - how do I do this?</a></li>
            <li><a href="#change_contact">My contact details have changed, how do I update them?</a></li>
        </ul>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>
        <h3>1. Using ArrayExpress</h3>
        <h5>Searching and understanding data</h5>
        <ul>
            <li><a id="search_help"></a><strong>How do I search the ArrayExpress database?</strong><br>
                For simple keyword search, please see the <a href="how_to_search.html">search help page</a> or try our
                <a href="/training/online/course/arrayexpress-discover-functional-genomics-data-qui/how-search-arrayexpress">interactive
                    free tutorial</a>. For more complex search queries, e.g. "Find all RNA-seq experiments studying
                diabetes in mouse", you will need to use some search filters to limit the search space of your keywords,
                e.g. where "diabetes" is the subject of investigation, and not merely mentioned in background
                information of a study. Please refer to the <a href="how_to_search.html#AdvancedSearchExperiment">advanced
                    search help</a> or this <a
                        href="/training/online/course/arrayexpress-discover-functional-genomics-data-qui/how-search-arrayexpress/advanced-search-fu">interactive
                    free tutorial</a>.
            </li>
        </ul>

        <ul>
            <li><a id="explain_browse_page"></a><strong>Can you guide me through the table of experiments I get after
                searching ArrayExpress?</strong><br>
                This
                <a href="/training/online/course/arrayexpress-discover-functional-genomics-data-qui/how-search-arrayexpress/results-table">online
                    training page</a> will get you started.
            </li>
        </ul>

        <ul>
            <li><a id="explain_magetab"></a><strong>How do I link sample and data file information in the downloadable
                files?</strong><br>
                Samples and data are linked together in the <a
                        href="/training/online/course/arrayexpress-discover-functional-genomics-data-qui/experiment-and-sample-information/sample-i">samples
                    table</a> on the ArrayExpress website. The table (called the SDRF, which stands for "Sample and Data
                Relationship Format") can be downloaded as a .txt spreadsheet, which will open nicely in any spreadsheet
                programme such as Microsoft Excel. To fully understand an experiment, we recommend that you check out
                also the <a
                        href="/training/online/course/arrayexpress-discover-functional-genomics-data-qui/experiment-and-sample-information">experiment
                    overview</a> too.
            </li>
        </ul>

        <ul>
            <li><a id="expt_already_published"></a><strong>I read a paper that referred to data in ArrayExpress but when
                I searched for the data I got no hits. Why?</strong><br>
                It could be that the submitter of the data has not informed us that the experiment should be made
                public. Please email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> with the
                literature citation where you spotted the ArrayExpress accession, and we will make the data public.
            </li>
        </ul>

        <ul>
            <li>
                <a id="acc_num_prefix"></a><strong>I have seen ArrayExpress experiment accessions with prefixes such
                as "E-MTAB", "E-GEOD", etc. What do the prefixes mean?<br>
            </strong>The prefixes indicate the source and/or submission route from which the data came from. The common
                ones are:
                <ul>
                    <li>MEXP = data submitted via the MIAMExpress submission tool (discontinued since July 2014)</li>
                    <li>TABM = data submitted via the Tab2MAGE submission tool (discontinued since January 2012)</li>
                    <li>MTAB = data submitted via the MAGE-TAB submission tool (discontinued since September 2014) or <a
                            href="/fg/annotare/login">Annotare</a></li>
                    <li>GEOD = data imported from <a href="https://www.ncbi.nlm.nih.gov/geo/">NCBI Gene Expression
                        Omnibus</a></li>
                </ul>
                See the <a href="accession_codes.html">accession codes</a> help page for more information.
            </li>
        </ul>
        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h5>Data access policy - permissions and restrictions</h5>
        <ul>
            <li><a id="data_restrictions"></a><strong>Are there any restrictions on the use of data obtained through
                ArrayExpress?</strong><br>
                No restrictions, all public data from ArrayExpress can be used by
                anyone and our services are completely free of charge.
            </li>
        </ul>

        <ul>
            <li><a id="need_login_or_not"></a><strong>Do I need a login account to view or access data in
                ArrayExpress?</strong><br>
                For public data, no login is required. All public data is available to everyone. For private,
                pre-published data, we provide login accounts to data submitters so that they, and their reviewers, can
                access them.
            </li>
        </ul>

        <ul>
            <li><a id="lost_login_details"></a><strong>I've lost my ArrayExpress website login details. What do I
                do?</strong><br>
                * Note: you do not need an ArrayExpress login for accessing public data. *
                If you are viewing a curated experiment which you submitted to ArrayExpress, use the
                <a href="how_to_search_private_data.html#LostPrivateDataLogin">forgotten password reminder form</a> in
                the login box.
                <p>If you are the reviewer, then please contact the journal editor.
                    We can only give ArrayExpress login details to the data depositor directly.</p></li>
        </ul>

        <ul>
            <li><a id="reviewer_need_access"></a><strong>I'm reviewing a paper. How do I get a login to view private
                data
                that the authors have deposited in ArrayExpress?</strong><br>
                Reviewer login details are sent to submitters on completion of the
                processing of their submission. Please contact the journal editor to request this login information.
                We cannot disclose login details to reviewers without first getting authorisation from the submitter or
                journal.
            </li>
        </ul>

        <ul>
            <li><a id="private_access"></a><strong>Will I have full access to an experiment while it is private in
                ArrayExpress?</strong><br>

                Yes, if the experiment is a microarray study. You can view the ArrayExpress record and download all
                associated files if you <a href="how_to_search_private_data.html">log in to ArrayExpress with a
                    data-access account</a>.<br>

                <p>No, if the experiment is a sequencing study. When you <a href="how_to_search_private_data.html"> log
                    in to ArrayExpress</a>, you will be able to see the full ArrayExpress record, and download processed
                    data files, but the links to raw read data files (in fastq format) are placeholders until the
                    experiment is made public. The reason is that raw data files are brokered to the <a href="/ena">European
                        Nucleotide Archive (ENA)</a>, which is part of the Sequence Read Archive (SRA), a collaboration
                    between ENA (EBI), Genbank (NCBI), and DDBJ (Japan). There is currently no infrastructure to access
                    privately held data files in SRA.</p>
            </li>
        </ul>


        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h5>FTP data download and programmatic access</h5>
        <ul>
            <li><a id="ftp_dl"></a><strong>Is there an FTP site to download data files in bulk?</strong><br>
                Yes, all the data and array designs in ArrayExpress are available for
                direct download. Please see <a href="FTP_archives.html">help on FTP downloads</a>.
            </li>
        </ul>

        <ul>
            <li><a id="standard_url"></a><strong>Is there a standard URL format for linking to an experiment in
                ArrayExpress?</strong><br>
                Yes, you can link to an experiment using its accession number. For
                example, to retrieve information about experiment E-MTAB-4567, use this
                link: <a href="/arrayexpress/experiments/E-MTAB-1730">https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1730</a>.
            </li>
        </ul>
        <ul>
            <li><a id="api"></a><strong>Do you have any application programming interfaces (APIs) for accessing
                ArrayExpress?</strong><br>
                Yes, we have a REST/JSON API. See <a href="programmatic_access.html">Programmatic access</a> for
                details.
            </li>
        </ul>
        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h5>Other general questions</h5>
        <ul>
            <li><a id="ae_geo_overlap"></a><strong>How much overlap is there between ArrayExpress and the NCBI Gene
                Expression
                Omnibus (GEO)?</strong><br>
                We import data on a weekly basis from <a href="https://www.ncbi.nlm.nih.gov/geo/"> NCBI Gene
                    Expression Omnibus (GEO)</a>. All experiments imported from GEO
                have accession numbers in the format of E-GEOD-n, where n is a number.
                For example, GEO accession "GSE29080" would become "E-GEOD-29080" in
                ArrayExpress. As of 6 Apr 2017, ArrayExpress contains 59285 GEO series (experiments),
                that is about 71% of the 83303 public series available from GEO on the same day.
                GEO currently does not mirror ArrayExpress experiments systematically.
                See the <a href="GEO_data.html">GEO data import page</a> for more details.
            </li>
        </ul>

        <ul>
            <li><a id="cite"></a><strong>How do I cite my ArrayExpress data sets in my publication?</strong><br>
                <p>You should include your experiment accession number and the URL to
                    ArrayExpress home page, <a href="/arrayexpress/">www.ebi.ac.uk/arrayexpress</a>.
                    e.g. "RNA-seq data have been deposited in the ArrayExpress database at EMBL-EBI (<a
                            href="/arrayexpress/">www.ebi.ac.uk/arrayexpress</a>)under accession number E-MTAB-xxxx."
                    You can also include a direct link to your experiment or array design. For example, the link would
                    be
                    <code>https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1234</code> for experiment
                    "E-MTAB-1234", or
                    <code>https://www.ebi.ac.uk/arrayexpress/arrays/A-MTAB-567</code> for array design "A-MTAB-567".</p>
                <p>If you wish to include a citation
                    for the ArrayExpress database, then the following publication should be used: Kolesnikov N. et al.
                    2015 ArrayExpress update - simplifying data submissions. Nucleic
                    Acids Res, doi: 10.1093/nar/gku1057. <a href="http://europepmc.org/abstract/MED/25361974">Pubmed ID
                        25361974</a>.</p></li>
        </ul>

        <ul>
            <li><a id="update_frequency"></a><strong>How often is ArrayExpress updated?</strong><br>
                The <a href="/arrayexpress/">public
                    web interface</a> and the <a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/">FTP
                    site</a>
                for ArrayExpress are updated daily at about 6am UK time.
            </li>
        </ul>
        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <!--START OF GENERAL SUBMISSION FAQ ANSWERS-->
        <p>&nbsp;</p>
        <h3>2. Submitting data to ArrayExpress</h3>
        <h5>Accession numbers</h5>
        <ul>
            <li>
                <a id="need_acc_num_urgent"></a><strong>I urgently need an accession number to include in my publication
                - how can I get one?</strong><br>
                Firstly, start your <a href="/arrayexpress/submit/overview.html">submission</a> if you have not already
                done so! ArrayExpress does not provide accession numbers in advance of
                submission of data. This is in line with the <a
                    href="http://www.ncbi.nlm.nih.gov/geo/info/faq.html#Whenaccessions">NCBI
                Gene Expression Omnibus (GEO)'s policy</a> of provision of accession numbers after deposition.
            </li>
        </ul>
        <ul>
            <li>
                <a id="how_long_to_get_acc"></a><strong>After I've submitted my experiment, how long will it take to get
                an accession number?</strong><p>
                On average, within 30 minutes. It all depends on the size of your data files, and how busy we are with
                submissions on a given day. Normally, you should receive an automated email with the accession number
                within 5 minutes (small microarray experiment) or up to 24 hours (large sequencing experiment). Please
                email us at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> , citing your experiment's title,
                if the automated email has not come through after a prolonged wait.</p>
            </li>
        </ul>
        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>


        <h5>Keeping unpublished data private</h5>
        <ul>
            <li><a id="keep_private"></a>
                <strong>Can my data be kept private after I submit it?</strong><br>
                Yes, all submitted information will be kept private
                until the release date that you set at the time of submission, or until
                a publication is released that contains the ArrayExpress accession
                number relating to your data. You can also change the release date to suit the
                peer review progress of your paper. Please see our <a href="data_availability.html#changes">data
                    availability</a>
                page for more information.
            </li>
        </ul>
        <ul>
            <li><a id="journal_private_access"></a>
                <strong>The journal I am submitting to has requested private access to my data - how do I get
                    an ArrayExpress login for them?</strong><br>
                Firstly, start your <a href="/arrayexpress/submit/overview.html">ArrayExpress submission</a> if you have
                not already
                done so! You will be sent login details for viewing your private experimental data
                on the <a href="/arrayexpress">ArrayExpress website</a>
                <em>only when curation is finished and when your data is loaded into the ArrayExpress database</em>. If
                you are really pushed
                for time and are struggling with your ArrayExpres submission, <a href="mailto:annotare@ebi.ac.uk">let us
                    know</a> and we will try our best to help.
            </li>
        </ul>

        <ul>
            <li><a id="double_blind"></a>
                <strong>The journal I am submitting to supports "double-blind" peer review. Can ArrayExpress hide my
                    name and contact details from the reviewers?</strong><br>
                Yes. If you're submitting a new data set via <a href="/fg/annotare/login">Annotare</a>, simply check the
                <code>Hide my identity from reviewers</code> box on the <code>Experiment Description</code> page. Please
                see <a href="/fg/annotare/help/describe_exp.html#remain_anonymous">submitter's guide on anonymity</a>
                for more information.<br>
                If you are the submitter of a previously non-anonymised private data set in ArrayExpress and would like
                to switch on anonymity, please write to us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>.
        </ul>


        <ul>
            <li><a id="private_login_problem"></a>
                <strong>I tried to login to ArrayExpress to see some recently
                    submitted private data but it said my
                    username and password were invalid. Why?</strong><br>
                Possibly you have tried to login with your submitter account with our submission tools, such as <a
                        href="/fg/annotare/login">Annotare</a>?
                These "submitter" accounts can only be used for
                submitting data, not viewing it in ArrayExpress. We will provide you
                with an ArrayExpress data access login account after your submission has been
                curated and loaded into ArrayExpress.
                <p>If you are using an
                    ArrayExpress login which you have just received, it might not be
                    working yet because ArrayExpress is updated only once a day at about
                    6am UK time. Try again after this time.</p>
                <p>If you've forgotten or lost
                    your access account details, you can retrieve it using the
                    <a href="how_to_search_private_data.html#LostPrivateDataLogin">forgotten password tool</a>. If it
                    still does not work, contact
                    us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
                    and tell us what login details you are trying to use / which page you
                    tried those details on.</p></li>
        </ul>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h5>Public release date</h5>
        <ul>
            <li><a id="what_date"></a>
                <strong>I don't know when my paper will be published so what
                    should I put as the release date?</strong><br>
                Enter an estimated public release date up to 1
                year in the future from the day of submission. The release date can be changed during/after curation for
                multiple times to match the peer-review progress. Avoid setting release date at public holidays (e.g.
                Christmas/New Year period) to avoid inadvertent release of your data. Please see our <a
                        href="data_availability.html#changes">release date change</a>
                page for more information.
            </li>
        </ul>
        <ul>
            <li><a id="make_public"></a>
                <strong>My paper is about to be published - how do I make the data public?</strong><br>
                You can <a href="data_availability.html#changes">change the release date</a> using this <a
                        href="https://www.ebi.ac.uk/fg/acext/">self-service tool</a>. Simply login with the data access
                account details which we emailed you when curation was completed, double-click at the private experiment
                that you would like release date changed, and follow on-screen instructions. You would normally have
                used these account details to log in to the ArrayExpress website and view your private experiment(s).
                Don't worry if you have forgotten the username and/or password, you can <a
                        href="how_to_search.html#Login">retrieve login details</a> using your submitter's email address
                and the experiment accession number. Your experiment will appear on the ArrayExpress website at about
                6am UK time the following day, after an overnight website update.
            </li>

        </ul>

        <ul>
            <li><a id="add_publication"></a>
                <strong>My paper has been published - how do I add the citation details to my experiment?</strong><br>
                You can add, remove or edit citation details for your experiment using this <a
                        href="https://www.ebi.ac.uk/fg/acext/">self-service tool</a>. Simply login with the data access
                account details which we emailed you when curation was completed, double-click at the experiment that
                you would like associate the publication with, and follow on-screen instructions. You would normally
                have used these account details to log in to the ArrayExpress website and view your private
                experiment(s).<br><br> Don't worry if you have forgotten the username and/or password, you can <a
                        href="how_to_search.html#Login">retrieve login details</a> using your submitter's email address
                and the experiment accession number. The changes you make will appear on the ArrayExpress website at
                about 6am UK time the following day, after an overnight website update.<br>
            </li>
        </ul>


        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h5>What data files to submit</h5>

        <ul>
            <li><a id="data_file"></a>
                <strong>Which data files do I need to submit for an experiment?</strong><br>
                The files required depend on the technology (e.g. microarray vs sequencing) and also the biological
                context of the study
                (e.g. single-cell RNA-seq requires some extra files). Please refer to <a
                        href="/arrayexpress/submit/overview.html">the submission overview page</a>, which will guide you
                through the different scenarios.
            </li>
        </ul>

        <ul>
            <li><strong><a id="mageml"></a>I have MAGE-ML/MAGE-TAB files generated from in-house pipelines. Does
                ArrayExpress
                still accept them?</strong><p>
                MAGE-ML files are no longer supported or accepted by ArrayExpress since 2011. MAGE-TAB spreadsheets,
                provided they are correctly formatted and <a href="ftp_upload.html">uploaded via FTP</a>, are still
                accepted in some exceptional cases under prior agreement with some submitters.</p>
                <p>The easiest way to submit data is to use our latest submission tool <a href="/fg/annotare/login">Annotare</a>,
                    which will generate MAGE-TAB files for you, so you don't have to worry about learning the file
                    format. All you need to do is to fill in a series of webforms and upload data files within the tool.
                    Annotare also has a built-in validator to save you writing validators/parsers to check your
                    submission.</p>
                <p>If you would like to update your pipeline to generate MAGE-TAB files, please write to us at <a
                        href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> to discuss further.</p>
            </li>
        </ul>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <h4>Making changes to data already submitted to ArrayExpress</h4>
        <ul>
            <li><strong><a id="hit_submit_too_soon"></a>I hit "Submit" at the end of my submission but I haven't
                actually finished editing.
                I logged in to the submission tool but can't edit my experiment anymore. Help!</strong><br>
                Email us at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> as soon as possible and ask us to
                "reopen" the submission and "assign" it back to you, so you can edit it again. (Submitted data sets are
                "read-only"
                to the submitter to avoid confusing, concurrent editing of the same submission by submitter and
                curator.) Remember to include your username and the experiment/array design title in your email so we
                can quickly find your submission.
            </li>
        </ul>
        <ul>
            <li><a id="data_already_in_AE_DB"></a><strong>My data has been loaded into ArrayExpress but I need to
                change/correct something - how do I do
                this?</strong><br>
                Email us at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> with the accession number and
                describe what corrections you would like
                to make. Please see this help page on <a href="modify_data_loaded_in_AE.html">updating experiments/array
                    designs</a> for further details.
            </li>
        </ul>
        <ul>
            <li><a id="change_contact"></a><strong>My contact details have changed, how do I update them?</strong><br>
                Send your new contact details to us at <a
                        href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
                and we will change them in ArrayExpress.
            </li>
        </ul>

        <p style="text-align: right;"><a href="#top" class="linktotop">Top</a></p>

        <!--END OF GENERAL SUBMISSION FAQ ANSWERS-->


    </section>
    <aside class="columns medium-3 last">
        <h3>Help</h3>
        <ul id="secondary-nav">
            <li><a href="index.html">Help home</a></li>
            <li><a href="quick_start.html">Quick Start guide</a></li>
            <li><a href="how_to_search.html">How to search</a>
                <ul>
                    <li><a href="how_to_search_adf.html">Array design search</a></li>
                    <li><a href="how_to_search_private_data.html">Search for private data</a></li>
                </ul>
            </li>
            <li><a href="submissions_overview.html">How to submit</a>
                <ul>
                    <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
                    <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                    <li><a href="single-cell_submission_guide.html">Single-cell submissions</a></li>
                    <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                    <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                    <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
                </ul>
            </li>
            <li><a href="data_availability.html">Learn more about our data</a>
                <ul>
                    <li><a href="data_availability.html">Data access policy</a></li>
                    <li><a href="accession_codes.html">Accession codes</a></li>
                    <li><a href="GEO_data.html">GEO data</a></li>
                    <li><a href="magetab_spec.html">MAGE-TAB format</a></li>
                </ul>
            </li>
            <li><a href="programmatic_access.html">Other ways to get data</a>
                <ul>
                    <li><a href="programmatic_access.html">Programmatic access</a></li>
                    <li><a href="FTP_archives.html">FTP archives</a></li>
                    <li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
                </ul>
            </li>
            <li class="active"><a href="FAQ.html">FAQ</a></li>
            <li><a href="contact_us.html">Contact us</a></li>
        </ul>
    </aside>
</div>
</body>
</html>
