<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
    <title>FAQ &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
<div id="content" role="main" class="grid_24 clearfix">
<section id="ae-help" class="grid_18 alpha">
<h2>Frequently Asked Questions (FAQ)</h2>
<p>If you cannot find the answer to your question in this FAQ or other <a href="index.html">ArrayExpress help pages</a> then please email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>.</p>

<p>The FAQ is divided into two sections:</p>
<ol>
    <li><a href="#use_ae_faq">Using ArrayExpress</a></li>
    <li><a href="#submit_faq_general">Submitting data to ArrayExpress</a></li>
</ol>
<p><strong>Please note that from October 2013, the MIAMExpress tool is no longer maintained for experiment submissions. Please use the <a href="submissions_overview.html">MAGE-TAB spreadsheet tool</a> or <a href="/fg/annotare/login">Annotare webform tool</a> instead. Array Design submissions can still be submitted via MIAMExpress.</strong></p>

<p>&nbsp;</p>
<a id="use_ae_faq"></a>
<h3>1. Using ArrayExpress</h3>
<h5>Searching and understanding data</h5>
<ul>
    <li><a href="#search_help">How do I search the ArrayExpress database?</a></li>
    <li><a href="#explain_browse_page">Can you guide me through the table of experiments I get after searching ArrayExpress?</a></li>
    <li><a href="#explain_magetab">How do I link sample and data file information in the downloadable files?</a></li>
    <li><a href="#expt_already_published">I read a paper that referred to data in ArrayExpress but when I searched
        for the data I got no hits. Why?</a></li>
    <li><a href="#acc_num_prefix">I have seen ArrayExpress experiment accessions with prefixes such as
        "E-MTAB", "E-GEOD", etc. What do the prefixes mean?</a></li>
</ul>

<h5>Data access policy - permissions and restrictions</h5>
<ul>
    <li><a href="#data_restrictions">Are there any restrictions on the use of microarray data obtained through ArrayExpress?</a></li>
    <li><a href="#need_login_or_not">Do I need a login account to view or access data in ArrayExpress?</a></li>
    <li><a href="#lost_login_details">I've lost my ArrayExpress website login details. What do I do?</a></li>
    <li><a href="#reviewer_need_access">I'm reviewing a paper. How do I get a login to view private data
        that the authors have deposited in ArrayExpress?</a></li>
</ul>

<h5>FTP data download and programmatic access</h5>
<ul>
    <li><a href="#ftp_dl">Is there an FTP site to download data files in bulk?</a></li>
    <li><a href="#standard_url">Is there a standard URL format for linking to an experiment in ArrayExpress?</a></li>
    <li><a href="#api">Do you have any application programming interfaces (APIs) for accessing ArrayExpress?</a></li>
</ul>

<h5>Other general questions</h5>
<ul>
    <li><a href="#cite">How do I cite ArrayExpress in my publication?</a></li>
    <li><a href="#ae_geo_overlap">How much overlap is there between ArrayExpress and the NCBI Gene Expression Omnibus (GEO)?</a></li>
    <li><a href="#update_frequency">How often is ArrayExpress updated?</a></li>
</ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>
    <a id="submit_faq_general"></a>
<h3>2. Submitting data to ArrayExpress</h3>

<h5>Accession numbers</h5>
<ul>
    <li><a href="#need_acc_num_urgent">I urgently need an accession number to include in my publication - how can I get one?</a></li>
    <li><a href="#how_long_to_get_acc">How long will it take to get an accession number?</a></li>
</ul>

<h5>Keeping unpublished data private</h5>
<ul>
    <li><a href="#keep_private">Can my data be kept private after I submit it?</a></li>
    <li><a href="#journal_private_access">The journal I am submitting to have requested private access to my data -
        how do I get an ArrayExpress login for them?</a></li>
    <li><a href="#private_login_problem">I tried to login to ArrayExpress to see some recently submitted private
        data but it said my username and password were invalid. Why?</a></li>
</ul>

<h5>Public release date</h5>
<ul>
    <li><a href="#what_date">I don't know when my paper will be published so what should I put as the release date?</a></li>
    <li><a href="#make_public">My paper is about to be published - how do I make the data public?</a></li>
</ul>

<h5>Which files to submit</h5>
<ul>
    <li><a href="#mtab_template">Can I get a MAGE-TAB template spreadsheet for submitting experiment data?</a></li>
    <li><a href="#affy">Which files do I need to submit for an Affymetrix experiment?</a></li>
    <li><a href="#illumina">How do I submit an Illumina experiment?</a></li>
    <li><a href="#hts">How do I submit a high throughput sequencing (HTS) experiment?</a></li>
    <li><a href="#large_files">I have very large data files, do I need to upload them through the web interface?</a></li>
    <li><a href="#mageml">I have MAGE-ML files generated from in-house pipelines. Does ArrayExpress still accept them?</a></li>
</ul>

<h5>Making changes to data already submitted to Arrayexpress</h5>
<ul>
    <li><a href="#hit_submit_too_soon">I hit "Submit" at the end of my submission but I haven't actually finished editing. I logged in to the submission tool but can't edit my experiment anymore. Help!</a></li>
    <li><a href="#data_already_in_AE_DB">My data has been loaded into ArrayExpress but I need to change/correct something - how do I do this?</a></li>
    <li><a href="#change_contact">My contact details have changed, how do I update them?</a></li>
</ul>

<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>
<h3>1. Using ArrayExpress</h3>
<h5>Searching and understanding data</h5>
<ul>
    <li><a id="search_help"></a><strong>How do I search the ArrayExpress database?</strong><br>
        For simple keyword searches and browsing, please visit the <a href="/training/online/course/arrayexpress-exploring-functional-genomics-data-ar/how-search-archive">search help page</a>. For more complex search queries, e.g. "Find me all the microarray experiments on Agilent chip X studying diabetes in mouse", please refer to the <a href="how_to_search.html#AdvancedSearchExperiment">advanced search help</a>.</li>
</ul>

<ul>
    <li><a id="explain_browse_page"></a><strong>Can you guide me through the table of experiments I get after searching ArrayExpress?</strong><br>
        See this <a href="quick_start.html">data browser help page</a> and this <a href="/training/online/course/arrayexpress-exploring-functional-genomics-data-ar/how-search-archive">section on how to interpret results in EBI Train Online course</a>.</li>
</ul>

<ul>
    <li><a id="explain_magetab"></a><strong>How do I link sample and data file information in the downloadable files?</strong><br>
        All the downloadable files relating to an experiment are in MAGE-TAB
        format. Top level information about an experiment (title, authors,
        protocols etc) is in a file called the IDF (Investigation Definition
        Format). Sample, extract and hybridization information is in a file
        called the SDRF (Sample and Data Relationship Format) which also links
        to the protocol information in the IDF. The SDRF lists which data files
        are associated with each hybridization or sequencing assay. For more information on
        the MAGE-TAB format, please visit the <a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab/introduction-mage-tab-format">MAGE-TAB
            help page</a>. The IDF and SDRF components are described in
        greater detail on <a href="/training/online/course/arrayexpress-submitting-data-using-mage-tab/creating-mage-tab-document">this
            help page</a>.</li>
</ul>

<ul>
    <li><a id="expt_already_published"></a><strong>I read a
        paper that referred to data in ArrayExpress but when I searched for the data I got no hits. Why?</strong><br>
        It could be that the submitter of the data has not yet told us that the experiment should be made public. If you come across this problem
        please email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> - tell us which paper you found the reference in and the ArrayExpress accession number quoted if any. We will make this data publicly available.</li>
</ul>

<ul>
    <li>
        <a id="acc_num_prefix"></a><strong>I have seen ArrayExpress experiment accessions with prefixes such
        as "E-MTAB", "E-GEOD", etc. What do the prefixes mean?<br>
    </strong>The prefixes indicate the source and/or submission route from which the data came from. The common ones are:
        <ul>
            <li>MEXP = data submitted via the <a href="/miamexpress/login.html">MIAMExpress</a> submission route </li>
            <li>TABM = data submitted via the Tab2MAGE submission route (discontinued since January 2012)</li>
            <li>MTAB = data submitted via the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB</a> submission route</li>
            <li>GEOD = data imported from <a href="http://www.ncbi.nlm.nih.gov/geo/">NCBI Gene Expression Omnibus</a></li>
        </ul>
        See the <a href="accession_codes.html">accession codes</a> help page for more information.</li>
</ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

<h5>Data access policy - permissions and restrictions</h5>
<ul>
    <li><a id="data_restrictions"></a><strong>Are
        there any restrictions on the use of microarray data obtained through
        ArrayExpress?</strong><br>
        No restrictions, all public data from ArrayExpress can be used by
        anyone and our services are completely free of charge.</li>
</ul>

<ul>
    <li><a id="need_login_or_not"></a><strong>Do
        I need a login account to view or access data in ArrayExpress?</strong><br>
        For public data, no. We only provide login accounts to data submitters
        so that they, and their reviewers, can access pre-publication private
        data. All other data can be viewed by everyone.</li>
</ul>

<ul>
    <li><a id="lost_login_details"></a><strong>I've
        lost my ArrayExpress website login details. What do I do?</strong><br>
        If you are viewing a curated experiment which you submitted to ArrayExpress, use the
        <a href="/arrayexpress/help/how_to_search.html#Login">forgotten password reminder form</a> in the login box.
        If you are not the data submitter (e.g. a reviewer), then please contact the the submitter.
        We can only give ArrayExpress login details to the owners of the data directly.</li>
</ul>

<ul>
    <li><a id="reviewer_need_access"></a><strong>I'm reviewing a paper. How do I get a login to view private data
        that the authors have deposited in ArrayExpress?</strong><br>
        Reviewer login details are sent to submitters on completion of the
        processing of their submission. Please contact the data submitter, via
        the journal editor, to request this login information. We cannot
        provide access to private data to anyone without first getting
        authorization from the submitter or journal.</li>
</ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

<h5>FTP data download and programmatic access</h5>
<ul>
    <li><a id="ftp_dl"></a><strong>Is
        there an FTP site to download data files in bulk?</strong><br>
        Yes, all the data and array designs in ArrayExpress are available for
        direct download in a number of different formats. For more information
        on what files are available and how to access them see our <a href="FTP_archives.html">FTP files for download help</a>.</li>
</ul>

<ul>
    <li><a id="standard_url"></a><strong>Is
        there a standard URL format for linking to an experiment in
        ArrayExpress?</strong><br>
        Yes, you can link to an experiment using its accession number. For
        example, to retrieve information about experiment E-MEXP-700 use this
        link: <a href="/arrayexpress/experiments/E-MEXP-700">http://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-700</a>.</li>
</ul>
<ul>
    <li><a id="api"></a><strong>Do
        you have any application programming interfaces (APIs) for accessing
        ArrayExpress?</strong><br>
        Yes, we have REST-style and WebService APIs for accessing ArrayExpress. Details can be found here: <a href="programmatic_access.html">Programmatic access</a>.</li>
</ul>

<h5>Other general questions</h5>
<ul>
    <li><a id="ae_geo_overlap"></a><strong>How
        much overlap is there between ArrayExpress and the NCBI Gene Expression
        Omnibus (GEO)?</strong><br>
        We import data on a weekly basis from <a href="http://www.ncbi.nlm.nih.gov/geo/"> NCBI Gene
            Expression Omnibus (GEO)</a>. All experiments imported from GEO
        have accession numbers in the format of E-GEOD-n, where n is a number.
        For example, GEO accession "GSE29080" would become "E-GEOD-29080" in
        ArrayExpress. For more information see the <a href="GEO_data.html">GEO
            data import page</a>.</li>
</ul>

<ul>
    <li><a id="cite"></a><strong>How do I cite ArrayExpress in my publication?</strong><br>
        You should include your experiment accession number and the URL to
        ArrayExpress home page, <a href="/arrayexpress/">www.ebi.ac.uk/arrayexpress</a>.
        e.g. "Microarray data are available in the ArrayExpress database (<a href="/arrayexpress/">www.ebi.ac.uk/arrayexpress</a>)
        under accession number E-MEXP-12345." If you wish to include a citation
        for ArrayExpress then the following publication should be used: Rustici.G et al. 2013 ArrayExpress update - trends in database growth and links to data analysis tools. Nucleic
        Acids Res, doi: 10.1093/nar/gks1174. <a href="http://europepmc.org/abstract/MED/23193272">Pubmed ID 23193272</a>.</li>
</ul>

<ul>
    <li><a id="update_frequency"></a><strong>How often is ArrayExpress updated?</strong><br>
        The <a href="/arrayexpress/">public
            web interface</a> and the <a href="ftp://ftp.ebi.ac.uk/pub/databases/microarray/">FTP site</a>
        for ArrayExpress are updated daily at about 06:00 GMT.</li>
</ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

    <!--START OF GENERAL SUBMISSION FAQ ANSWERS-->
<p>&nbsp;</p>
<h3>2. Submitting data to ArrayExpress (general)</h3>
<h5>Accession numbers</h5>
<ul>
    <li>
        <a id="need_acc_num_urgent"></a><strong>I urgently need an accession number to include in my publication - how can I get one?</strong><br>
        Firstly, start your submission if you have not already done so! ArrayExpress does not provide accession numbers in advance of
        submission of data. This is in line with the <a href="http://www.ncbi.nlm.nih.gov/geo/info/faq.html#Whenaccessions">NCBI
        Gene Expression Omnibus (GEO)'s policy</a> of provision of accession numbers after deposition. If you have completed the
        submission and are waiting for a response you can email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
        to let us know how urgently the accession number is required. To help us find your submission tell us your username, the title of the
        experiment or array. We will reply to your email as quickly as possible but please note that we often have several submissions requiring urgent
        attention. </li>
</ul>
<ul>
    <li>
        <a id="how_long_to_get_acc"></a><strong>How long will it take to get an accession number?<br>
    </strong>If you submit using the <a href="/cgi-bin/microarray/magetab.cgi"> MAGE-TAB spreadsheet tool</a>, after you complete a submission it is put in a
        queue awaiting review and checking by a curator. We aim to respond within a week upon receiving the submission, but response times do vary 
        depending on submission volume. If there are no major issues with your submission (e.g. there is adequate meta-data in the 
        MAGE-TAB spreadsheet describing your experiment and samples, all required data files are correctly formatted), the curator will send you the
        accession number, otherwise he/she will contact you to advise on how to revise and resubmit before an accession number can be assigned. Therefore,
         assignment of accession numbers also depends on how well the submitted files are prepared.
        <br><br>
        <p>If you submit using <a href="/fg/annotare/login">Annotare</a>,
        after you've filled in the webforms with experimental information and uploaded data files, all information will go through a mandatory validation step. If there are no errors,
        you can proceed to submit and will then receive an automatic confirmation email containing the accession number.</p></li>
</ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>


<h5>Keeping unpublished data private</h5>
<ul>
    <li><a id="keep_private"></a>
        <strong>Can my data be kept private after I submit it?</strong><br>
        Yes, all submitted information will be kept private
        until the release date that you set at the time of submission, or until
        a publication is released that contains the ArrayExpress accession
        number relating to your data. You can also change the release date to suit the 
        peer review progress of your paper.  Please see our <a href="data_availability.html">data availability</a>
        page for more information.</li>
</ul>
<ul>
    <li><a id="journal_private_access"></a>
        <strong>The journal I am submitting to have requested private access to my data - how do I get
            an ArrayExpress login for them?</strong><br>
        Firstly, start your ArrayExpress submission if you have not already
        done so! You will be sent login details for viewing your private experimental data
        on the <a href="/arrayexpress">ArrayExpress website</a>
        only when curation is finshed and when your data is loaded into the ArrayExpress database. If you are really pushed
        for time, reply to the email thread sent to you by the ArrayExpress
        curator in charge of your submission, or email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
        to let us know about your situation. We will try our best to fast-track
        your submission.</li>
</ul>
<ul>
    <li><a id="private_login_problem"></a>
        <strong>I tried to login to ArrayExpress to see some recently
            submitted private data but it said my
            username and password were invalid. Why?</strong><br>
        Possibly you have tried to login with your submitter account with our submission tools (
        <a href="/fg/annotare/login">Annotare</a>,
        <a href="/cgi-bin/microarray/magetab.cgi"> MAGE-TAB</a>,
        or <a href="/miamexpress/login.html"> MIAMExpress</a>)?
        These "submitter" accounts can only be used for
        submitting data, not viewing it in ArrayExpress. We will provide you
        with an ArrayExpress data access login account after your submission has been
        curated and then loaded into ArrayExpress. If you are using an
        ArrayExpress login which you have just received, it might not be
        working yet because ArrayExpress is updated only once a day at about
        06:00 GMT. Try again after this time.
        <br><br>
        <p>If you've forgotten or lost
        your access account details, you can retrieve it using the 
        <a href="how_to_search.html#Login">forgotten password tool</a>. If it still does not work, contact
        us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
        and tell us what login details you are trying to use / which page you
        tried those details on.</p></li>
</ul>

<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

<h5>Public release date</h5>
<ul>
    <li><a id="what_date"></a>
        <strong>I don't know when my paper will be published so what
            should I put as the release date?</strong><br>
        Enter an estimated public release date up to 1
        year in the future from the day of submission. The release date can be changed during/after curation for multiple times to match the peer-review progress. Please see our <a href="data_availability.html#changes">release date change</a>
        page for more information.</li>
</ul>
<ul>
    <li><a id="make_public"></a>
        <strong>My paper is about to be published - how do I make the data public?</strong><br>
        Email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> 
        with the experiment accession number and tell us when it should be made
        public. Please send us the citation information, including PubMed ID
        and Digital object identifier (DOI) too so we can add this to your
        experiment's record. Please see our <a href="data_availability.html">data
            availability</a> page for more information.</li></ul>
<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

<h5>What files to submit</h5>
<ul>
    <li><a id="mtab_template"></a>
        <strong>Can I get a MAGE-TAB template spreadsheet for submitting experiment data?</strong><br>
        Yes. Using the <a href="/cgi-bin/microarray/magetab.cgi"> MAGE-TAB spreadsheet submission
            tool</a>, you can generate a template spreadsheet specific to your experiment.
        Check out our <a href = "/fgpt/magetab/help/quick_start.html">
        MAGE-TAB quick start submission guide</a> or <a href="http://youtu.be/Ixtd4JfBV8Q">YouTube video</a>
        to see how the submission tool works.<br><br>
        If you use the <a href="/fg/annotare/login">Annotare</a> webform submission tool, there is no need to
        create any MAGE-TAB spreadsheet. Just fill in the webforms step-by-step, and a MAGE-TAB spreadsheet
        will be constructed for you, based on what you've entered on the webforms. See this
        <a href="/fgpt/annotare_help/">Annotare help page</a> for more information.
        <br><br>
        There are also some third party data annoation tools that can generate MAGE-TAB spreadsheets for you
        for uploading through the MAGE-TAB spreadsheet submission tool. There
        is more information about them <a href="third_party_tools.html">here</a>.</li>
</ul>
<ul>
    <li><a id="affy"></a>
        <strong>Which files do I need to submit for an Affymetrix
            experiment?</strong><br>
        For a <a href="http://www.mged.org/Workgroups/MIAME/miame_2.0.html">MIAME
            compliant</a> submission we need Affymetrix .CEL files as raw
        data and some form of processed, probe set level data. The processed
        data is preferably a matrix file generated by software such as as <a href="http://biostatistics.oxfordjournals.org/content/4/2/249.full.pdf">robust
            multi-array average (RMA)</a> or <a href="http://biosun1.harvard.edu/complab/dchip/">dChip</a>.
        A matrix file has the Affyemtrix probesets in the rows and processed
        intensity values per hybridisation in the columns <a href="/fgpt/magetab/help/data_format.html#data_matrix">
            (MAGE-TAB data matrix example)</a>.</li>
</ul>
<ul>
    <li><a id="illumina"></a>
        <strong>How do I submit an Illumina experiment?</strong><br>
        You do not need to submit data for every spot
        on the array. Please use the <a href="http://www.illumina.com/software/genomestudio_software.ilmn">Illumina
            GenomeStudio software </a>(previously known as the "BeadStudio"
        software) with the 'ArrayExpress Data Submission Report Plug-in' to
        generate the raw data file for submission. The plug-in is available
        from the <a href="http://www.illumina.com/software/illumina_connect.ilmn#gene_expression_plug_ins">Illumina
            connect website</a>. Please take a look at this <a href="/fgpt/magetab/help/data_format.html#illumina">example
            Illumina raw data file</a>. The unique identifiers in the raw
        data file should be taken from the "PROBE_ID" column of the
        GenomeStudio-generated file. (Note: the Illumina GenomeStudio-generated file would have column headings
        "PROBE_ID" and "ProbeID", and they are not the same! The latter often corresponds to "array address ID" 
        in the gene chip manifest file and it's the former that we're after.)
        <br><br>
        For processed data, a data matrix is preferred (PROBE_IDs in rows, processed measurements per hybridisation
        in columns), as in this <a href="/fgpt/magetab/help/data_format.html#data_matrix">MAGE-TAB matrix file example</a>. </li>
</ul>
<ul>
    <li><strong><a id="hts"></a>How do I submit a high throughput sequencing (HTS) experiment?</strong><br>
        Whichever submission tool you may use (<a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB</a> or <a href="/fg/annotare/login">Annotare</a>), 
        you will need to provide as much information as possible about the experiment's purpose, starting materials (samples),
        sequencing libraries, wet- and dry-lab protocols used, raw sequence data files for each sequencing
        assay, and optionally processed data files, e.g. BAM alignment files, RPFM/FPKM values for each gene/transcript.
        ArrayExpress acts as a data broker and will transfer the raw read files to the <a href="/ena">European Nucleotide Archive (ENA)</a>
        on your behalf, so you do not need to send the read files directly to the ENA.
        <br><br>
        Raw data files must be in a format <a href="/ena/about/read-file-formats">accepted by the Sequence Read Archive (SRA)</a> at the ENA,
        otherwise we will not be able to process your submission.  We strongly recommend that you check out our <a href="UHTS_submissions.html">sequencing
        submissions help page</a> as it explains the requirements in a lot more detail. </li>
</ul>

<ul>
    <li><strong><a id="large_files"></a>I have very large data files, do I need to upload them through the web interface?</strong><br>
        Large (over 10Mb) data files can be sent to us via <a href="ftp_upload.html"> FTP</a> instead of uploaded through the <a href="/cgi-bin/microarray/magetab.cgi">MAGE-TAB spreadsheet</a>
         or <a href="/fg/annotare/login">Annotare webform</a> tool. If you're submitting using MAGE-TAB tool, please email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> to let us know if you have transferred the files. (Note: When using MAGE-TAB tool, one small size (dummy/toy) data file must also be uploaded to allow the submission to be completed. This dummy file will be ignored as long as it is not listed in the MAGE-TAB meta-data spreadsheet.)
    </li>
</ul>


<ul>
    <li><strong><a id="mageml"></a>I
        have MAGE-ML files generated from in-house pipelines. Does ArrayExpress
        still accept them?</strong><br>
        No. MAGE-ML files are no longer supported and accepted by ArrayExpress.
        Please submit your array design files and/or experiment data using <a href="/cgi-bin/microarray/magetab.cgi">
        MAGE-TAB</a> or <a href="/fg/annotare/login">Annotare</a>, which are described on this <a href="submissions_overview.html">submissions overview page</a>.
        If you would like to update your pipeline to generate MAGE-TAB files, please write to us at
        <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> and we will try our best to offer assistance.
    </li>
</ul>

<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

<h4>Making changes to data already submitted to ArrayExpress</h4>
<ul>
    <li><strong><a id="hit_submit_too_soon"></a>I hit "Submit" at the end of my submission but I haven't actually finished editing.
        I logged in to the submission tool but can't edit my experiment anymore. Help!</strong><br>
        Email us at <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a> (MAGE-TAB submitters) or
        at <a href="mailto:annotare@ebi.ac.uk">annotare@ebi.ac.uk</a> (Annotare submitters) 
        as soon as possible and ask us to "reopen" the submission or to "assign" it back to you. (Once submitted, your submission
        may already be worked on by a curator and is considered "closed", so as to avoid confusing,
        concurrent editing of the same submission by you and the curator.) Remember to include your username 
        and the experiment/array design title in your email so we can quickly find your submission. 
        </li>
</ul>
<ul>
    <li><a id="data_already_in_AE_DB"></a><strong>My
        data has been loaded
        into ArrayExpress but I need to change/correct something - how do I do
        this?</strong><br>
        Email us at
        <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
        with the accession number and describe what corrections you would like
        to make. Minor corrections such as fixing typing errors can be made
        easily. Major changes like adding or removing samples/assays usually
        require the experiment to be unloaded from the ArrayExpress database, 
        edited, re-curated and reprocessed, which can take a considerable amount
        of time.<br><br>
        <i>Note on editing public sequencing experiment:</i> Due to data mirroring agreement between all partners of
        the International Nucleotide Sequence Database Collaboration (INSDC, involving 
        <a href="http://www.ddbj.nig.ac.jp">DDBJ</a>, <a href="/ena">ENA</a>, and 
        <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>), 
        once a sequencing data set has been released to the public, it is INSDC's policy <i>not</i> 
        to make any changes to the data set (e.g. turning it private again, adding/removing/changing
        samples), unless there is an exceptional reason. As ArrayExpress <a href="uhts_submissions/#DataTypes">
        brokers sequencing submission to the ENA</a>, to make sure experimental data remains consistent
        across ENA and ArrayExpress, you will not be allowed to change/correct the ArrayExpress records 
        for your public sequencing experiment either.
        </li>
</ul>
<ul>
    <li><a id="change_contact"></a><strong>My
        contact details have
        changed, how do I update them?</strong><br>
        Send your new contact details to us at
        <a href="mailto:arrayexpress@ebi.ac.uk">arrayexpress@ebi.ac.uk</a>
        and we will change them in ArrayExpress.</li>
</ul>

<p style="text-align: right;"><a href="#top" class="linktotop">Top</a>

    <!--END OF GENERAL SUBMISSION FAQ ANSWERS-->


</section>
<aside class="grid_6 omega">
    <h3>Help</h3>
    <ul id="secondary-nav">
        <li><a href="index.html">Help home</a></li>
        <li><a href="quick_start.html">Quick Start guide</a></li>
        <li><a href="how_to_search.html">How to search</a></li>
        <li><a href="programmatic_access.html">Other ways to get data</a>
            <ul>
                <li><a href="programmatic_access.html">Programmatic access</a></li>
				<li><a href="FTP_archives.html">FTP archives</a></li>
				<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
            </ul>
        </li>
        <li><a href="submissions_overview.html">How to submit</a>
            <ul>
                <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                <li><a href="third_party_tools.html">Third-party tools</a></li>
            </ul>
        </li>
        <li class="active"><a href="FAQ.html">FAQ</a></li>
        <li><a href="data_availability.html">Learn more about our data</a>
            <ul>
                <li><a href="data_availability.html">Data access policy</a></li>
                <li><a href="accession_codes.html">Accession codes</a></li>
                <li><a href="GEO_data.html">GEO data</a></li>
            </ul>
        </li>
        <li><a href="contact_us.html">Contact us</a></li>
    </ul>
</aside>
</div>
</body>
</html>
