<!DOCTYPE html>
<html class="no-js" lang="en">
<head><title>FTP Archives &lt; Help &lt; ArrayExpress &lt; EMBL-EBI</title></head>
<body class="level2">
    <div id="content" role="main" class="columns medium-12 clearfix">
        <section id="ae-help" class="columns medium-9">
            <h2>FTP Archives<br></h2>
            <p>Files (metadata and data) for public experiments and array designs in ArrayExpress can be downloaded from our public FTP site.</p>
            <p>The FTP address is <a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/">ftp.ebi.ac.uk/pub/databases/arrayexpress/data/</a> </p>
            <h3>How are the files organised?</h3>
            <p>The FTP site consists of two major directories:<br></p>
            <table>
              <thead>
                <tr>
                  <th>Directory</th>
                  <th>Content<th>
                </tr>
              </thead>
                <tbody>
                <tr>
                    <td><a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/array/">array</a> </td>
                    <td> Descriptions of all array designs in ArrayExpress. </td>
                </tr>
                <tr>
                    <td><a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/experiment/">experiment</a> </td>
                    <td> Data files and meta data (sample descriptions, protocols, etc) for all experiments in ArrayExpress. </td>
                </tr>
                </tbody>
            </table>
            <p>Under each directory, files are grouped per experiment or per array design,
                first by the four letter code and then the accession number. </p>
            <p>For example:</p>
            <ul>
                <li>Files for array design A-AFFY-23 <br><a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/array/AFFY/A-AFFY-23/"> ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/array/AFFY/A-AFFY-23/</a></li>
            </ul>
            <ul>
                <li>Files for experiment E-MEXP-440 <br><a href="ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/experiment/MEXP/E-MEXP-440/">ftp://ftp.ebi.ac.uk/pub/databases/arrayexpress/data/experiment/MEXP/E-MEXP-440/</a></li>
            </ul>
            <p>&nbsp;</p>
            <h3>File types in experiment directories</h3>
            <p>The experiment information (meta data) and data files are available in a number of different formats.</p>
            <table>
                <thead>
                <tr>
                    <th>File extension</th>
                    <th>Description</th>
                </tr>
              </thead>
              <tbody>
                <tr>
                    <td> .idf.txt</td>
                    <td> Top level information about the experiment including title, description, submitter contact details and protocols.</td>
                </tr>
                <tr>
                    <td>.sdrf.txt</td>
                    <td>Information about the samples, the relationships between the samples, extracts,
                        labeled extracts, hybridizations, factor values and data files.</td>
                </tr>
                <tr>
                    <td>.raw.1.zip</td>
                    <td>All raw data files as named in the .sdrf.txt file (see above). <br>
                        Files are in .CEL format and generic tab-delimited format for
                        Affymetrix and non-Affymetrix experiments respectively. Large data sets
                        are split into a number of different archives with extensions such as
                        ".raw.1.zip", ".raw.2.zip", etc. </td>
                </tr>
                <tr>
                    <td>.processed.1.zip</td>
                    <td>All processed data files as named in the .sdrf description file.&nbsp; (Not all experiments have processed data files.)<br>
                        Files are in generic tab delimited text format. Each processed file may
                        contain data from 1 or more hybridisations/assays.&nbsp; Similar to raw
                        data files, large data sets are split into a number of different
                        archives with extensions such as ".processed.1.zip",
                        ".processed.2.zip", etc.
                </tr>
                <tr>
                    <td>.bam, .bam.bai, .bam.prop</td>
                    <td>For a subset of highly-curated RNA-seq experiments in ArrayExpress, BAM
                        alignment files are avialable. They are generated by the ArrayExpress
                        team by processing the experimental raw FASTQ data files with the <a href="http://bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html">ArrayExpressHTS BioConductor package</a> on the <a href="/Tools/rcloud">EBI R Cloud workbench</a>.
                    </td>
                </tr>
                <tr>
                    <td>.eSet.r</td>
					<td>For some experiments, ArrayExpress team generated R ExpressionSet objects using version 2.11 of the <a href="http://bioconductor.org/packages/release/bioc/html/ArrayExpress.html">ArrayExpress BioConductor package</a>. The R object can be easily loaded into Bioconductor for downstream analysis. See the "<a href="expressionset_objects.html">help page</a> and the ArrayExpress package <a href="http://www.bioconductor.org/packages/release/bioc/vignettes/ArrayExpress/inst/doc/ArrayExpress.pdf">vignette</a> for more details on using these objects.</td>
                </tr>
                <tr>
                    <td>additional.1.zip</td>
                    <td>Additional files provided by the experiment's submitter which are
                        supplementary to the IDF, SDRF, raw data and processed data
                        files.&nbsp; The format and content of these files are not controlled.<br>
                    </td>
                </tr>
                </tbody>
            </table>
            <p>For some experiments there are some legacy files such as MAGE-ML files still available on the FTP site.</p>
            <p>&nbsp;</p>
            <h3>File types in array directories</h3>
            <table>
                <thead>
                <tr>
                    <th>File extension</th>
                    <th>Description</th>
                </tr>
              </thead>
                <tbody>
                <tr>
                    <td>.adf.txt</td>
                    <td> "adf" stands for "array design file". This file captures all the annotation about the probes on the array,
                        e.g. the unique probe identifiers, the coordinate location of the probe on the microarray, genomic mapping
                        location of each probe, whether a probe is in the experimental or control set, what kind of control probe
                        it is (negative control, empty spot with buffer only, etc).</td>
                </tr>
                <tr>
                    <td>.features.txt<br>.mageml.tar.gz<br>.reporters.txt</td>
                    <td>These are legacy files in MAGE-ML format, which
                        was previously used in ArrayExpress (until mid-2011), but no longer supported in ArrayExpress after switching
                        to <a href="http://www.fged.org/projects/mage-tab/">MAGE-TAB</a> format.  All .adf.txt files (see description above) are
                        now in MAGE-TAB format.  MAGE-ML files are included for some arrays for the sake of completeness.</td>
                </tr>
                </tbody>
            </table>
        </section>
        <aside class="columns medium-3 last">
            <h3>Help</h3>
    <ul id="secondary-nav">
        <li><a href="index.html">Help home</a></li>
        <li><a href="quick_start.html">Quick Start guide</a></li>
        <li><a href="how_to_search.html">How to search</a>
            <ul>
                <li><a href="how_to_search_adf.html">Array design search</a></li>
                <li><a href="how_to_search_private_data.html">Search for private data</a></li>
            </ul>
        </li>
        <li><a href="submissions_overview.html">How to submit</a>
            <ul>
                <li><a href="pre-submission_checklist.html">Pre-submission checklist</a></li>
                <li><a href="UHTS_submissions.html">Sequencing submissions</a></li>
                <li><a href="single-cell_submission_guide.html">Single-cell submissions</a></li>
                <li><a href="adf_submissions_overview.html">Array design submissions</a></li>
                <li><a href="ftp_upload.html">FTP upload instructions</a></li>
                <li><a href="modify_data_loaded_in_AE.html">Modify data in ArrayExpress</a></li>
            </ul>
        </li>
        <li><a href="data_availability.html">Learn more about our data</a>
            <ul>
                <li><a href="data_availability.html">Data access policy</a></li>
                <li><a href="accession_codes.html">Accession codes</a></li>
                <li><a href="GEO_data.html">GEO data</a></li>
                <li><a href="magetab_spec.html">MAGE-TAB format</a></li>
            </ul>
        </li>
        <li><a href="programmatic_access.html">Other ways to get data</a>
            <ul>
                <li><a href="programmatic_access.html">Programmatic access</a></li>
				<li class="active"><a href="FTP_archives.html">FTP archives</a></li>
				<li><a href="expressionset_objects.html">ExpressionSet R objects</a></li>
            </ul>
        </li>
        <li><a href="FAQ.html">FAQ</a></li>
        <li><a href="contact_us.html">Contact us</a></li>
    </ul>
        </aside>
    </div>
</body>
</html>
